### PR TITLE
Hwankyus slide - pull request

### PR DESCRIPTION
**Pull request by hwankyusong**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,7 +32,18 @@ <h3>Your name</h3>
 		  	     </ul>
 		    </section>
 		    <section>
+        hwankyus_slide
```
Comment (shots47s) [inline index.html]: Extraneous statement

File: index.html [hunk 2]
```diff
@@ -32,7 +32,18 @@ <h3>Your name</h3>
...
+				<h3>Hwankyu Song</h3>
+			    <ul>
+		              <li> Bayside, NY
+			      <li> Data Analytics for Science
+		              <li> Good Will Hunting
+		  	     </ul>
+
+
 			   copy above and fill in
+        main
```
Comment (shots47s) [inline index.html]: Extraneous statement

File: index.html [hunk 3]
```diff
@@ -32,7 +32,18 @@ <h3>Your name</h3>
...
+
+				<h3>Hwankyu Song</h3>
+			    <ul>
+		              <li> Bayside, NY
+			      <li> Data Analytics for Science
+		              <li> Good Will Hunting
+		  	     </ul>
+
+
 			   copy above and fill in
```
Comment (shots47s) [inline index.html]: Remove this line.

**Review by shots47s — CHANGES_REQUESTED** [review]
Please see my comments in the file to correct, you have several extraneous statements.

**Comment by hwankyusong** [comment]
I believe I've removed all the extraneous statements, as per your comments.

**Review by shots47s — APPROVED** [review]
Looks good.

**Pull request merged**